### PR TITLE
[docs] Update main README

### PR DESCRIPTION
**Pull request by ruizb**
### What and why?

* Update the main README:
   * Add a step to the release process to update the CI integrations if changes to the `synthetics` command are released
   * Move the "Usage" section (with the links to other READMEs) up in the file
   * A few nits here and there
* Remove the checkbox specific to `synthetics` changes in the PR template

### CODE REVIEW

**Comment by ruizb** [comment]
For @DataDog/documentation and @peculiarism: Do you think we should update the first sentences of the repo? Right now it says:

> [...] It currently features running synthetics tests and waiting for the results.

But it's been a while since `datadog-ci` can do more than that.

File: README.md [hunk 1]
```diff
@@ -108,7 +51,7 @@ Available commands:
 
 Each command allows interacting with a product of the Datadog platform. The commands are defined in the [src/commands](/src/commands) folder.
 
-Further documentation for each command can be found in its folder, ie:
+Further documentation for these commands can be found in their respective folders:
```
Comment (Drarig29) [inline README.md:54]: Maybe we could merge the "Available commands" list in the Usage just above with the documentation of each of the commands?

```bash
Usage: datadog-ci <command> <subcommand> [options]
```

Possible values for `<command>`:
- `dsyms` (see [iOS dSYM Files](src/commands/dsyms/))
- `flutter-symbols` (see [Flutter Symbols](src/commands/flutter-symbols/))
- `git-metadata` (see [Git metadata](src/commands/git-metadata))
- `junit` (see [JUnit XML](src/commands/junit))
- `lambda` (see [Lambda](src/commands/lambda))
- `metric` (see [Metric](src/commands/metric))
- `react-native` (see [React Native sourcemaps](src/commands/react-native/))
- `sourcemaps` (see [Browser sourcemaps](src/commands/sourcemaps/))
- `synthetics` (see [Synthetics CI/CD Testing](src/commands/synthetics/))
- `tag` (see [Tag](src/commands/tag))
- `trace` (see [Trace](src/commands/trace))

**Comment by buraizu** [comment]
Thanks, created DOCS-4228 for tracking

File: README.md [hunk 2]
```diff
@@ -29,98 +29,27 @@ npm install -g @datadog/datadog-ci
...
-cd container
-docker build --tag datadog-ci .
-```
-
-Optionally you can use the `VERSION` build argument to build an image for a specific version:
-
-```sh
-docker build --build-arg "VERSION=v1.14" --t datadog-ci .
-```
+For more ways to install the CLI, please refer to [this section](#more-ways-to-install-the-cli).
```
Comment (alai97) [inline README.md:32]:
```suggestion
For more ways to install the CLI, see [this section](#more-ways-to-install-the-cli).
```

File: README.md [hunk 3]
```diff
@@ -29,98 +29,27 @@ npm install -g @datadog/datadog-ci
...
 
 ## Usage
 
 ```bash
 Usage: datadog-ci <command> <subcommand> [options]
+```
+
+Possible values for `<command>` and their documentation:
```
Comment (alai97) [inline README.md:40]:
```suggestion
Possible values for each `<command>` and corresponding documentation include the following:
```

File: README.md [hunk 4]
```diff
@@ -187,7 +116,7 @@ The CI performs tests to avoid regressions by building the project, running unit
 
 The end-to-end test installs the package in a new project, configures it (using files in the `.github/workflows/e2e` folder) and runs a `synthetics run-tests` command in a Datadog Org (`Synthetics E2E Testing Org`) to verify the command is able to perform a test.
```
Comment (alai97) [inline README.md:117]:
```suggestion
The end-to-end test installs the package in a new project, configures it by using files in the `.github/workflows/e2e` folder, and runs a `synthetics run-tests` command in a Datadog org (such as `Synthetics E2E Testing Org`) to verify the command is able to perform a test.
```

File: README.md [hunk 5]
```diff
@@ -187,7 +116,7 @@ The CI performs tests to avoid regressions by building the project, running unit
...
 
-The synthetics tests ran are a browser test (id `neg-qw9-eut`) and an API test (id `v5u-56k-hgk`), both loading a page which outputs the headers of the request and verifying the `X-Fake-Header` header is present. This header is configured as an override in the `.github/workflows/e2e/test.synthetics.json` file. The API and Application keys used by the command are stored in Github Secrets named `datadog_api_key` and `datadog_app_key`.
+The synthetics tests ran are a browser test (id `neg-qw9-eut`) and an API test (id `v5u-56k-hgk`), both loading a page which outputs the headers of the request and verifying the `X-Fake-Header` header is present. This header is configured as an override in the `.github/workflows/e2e/test.synthetics.json` file. The API and Application keys used by the command are stored in GitHub Secrets named `datadog_api_key` and `datadog_app_key`.
```
Comment (alai97) [inline README.md:119]:
```suggestion
The Synthetic tests that are run include a browser test (with a test ID of `neg-qw9-eut`) and an API test (with a test ID of `v5u-56k-hgk`). Both tests load a page which outputs the headers of the request and verifies the `X-Fake-Header` header is present. This header is configured as an override in the `.github/workflows/e2e/test.synthetics.json` file. The API and application keys used by the command are stored in GitHub Secrets named `datadog_api_key` and `datadog_app_key`.
```

File: README.md [hunk 6]
```diff
@@ -214,30 +143,95 @@ yarn prepack
 
 Releasing a new version of `datadog-ci` unfolds as follow:
```
Comment (alai97) [inline README.md:144]:
```suggestion
To release a new version of `datadog-ci`:
```

File: README.md [hunk 7]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
-- push the branch along with the tag to the upstream (Github), create a Pull Request with the changes introduced detailed in the description and get at least one approval. ([sample Pull Request](https://github.com/DataDog/datadog-ci/pull/78))
-- merge the Pull Request
-- create a Github Release from the [Tags page](https://github.com/DataDog/datadog-ci/tags) with the description of changes introduced
-- Once the release has been created, a Github Action will publish the package and a Gitlab pipeline will publish the Docker image. Make sure they succeed.
+1. Create a new branch for the version upgrade.
+2. Update the version using `yarn version [--patch|--minor|--major]`, depending on the nature of the changes introduced. You may refer to [Semantic Versioning](https://semver.org/#summary) to determine which to increment.
+3. Push the branch along with the tag to the upstream (GitHub) with `git push --tags origin name-of-the-branch`, then create a Pull-Request with the changes introduced detailed in the description, and get at least one approval. ([sample Pull-Request](https://github.com/DataDog/datadog-ci/pull/78))
```
Comment (alai97) [inline README.md:148]:
```suggestion
3. Push the branch along with the tag to the upstream (GitHub) with `git push --tags origin name-of-the-branch`, create a Pull Request with the changes introduced in the description details, and get at least one approval. For example, see this [sample pull request](https://github.com/DataDog/datadog-ci/pull/78).
```

File: README.md [hunk 8]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+4. Merge the Pull-Request.
```
Comment (alai97) [inline README.md:149]:
```suggestion
4. Merge the Pull Request.
```

File: README.md [hunk 9]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+5. Create a GitHub Release from the [Tags page](https://github.com/DataDog/datadog-ci/tags) with the description of changes introduced.
+6. Once the release has been created, a GitHub Action will publish the package, and a Gitlab pipeline will publish the Docker image. Make sure they succeed.
```
Comment (alai97) [inline README.md:151]:
```suggestion
6. Once the release has been created, a GitHub Action publishes the package, and a Gitlab pipeline publishes the Docker image. Make sure these jobs succeed.
```

File: README.md [hunk 10]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+7. :warning: If the release introduced some **changes in the `synthetics` command**, you have to upgrade `datadog-ci` in the following projects:
```
Comment (alai97) [inline README.md:152]:
```suggestion
7. If the release introduced some **changes in the** `synthetics` **command**, you have to upgrade `datadog-ci` in the following projects:
```

File: README.md [hunk 11]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+   * [GitHub Action](https://github.com/DataDog/synthetics-ci-github-action)
+   * [CircleCI Orb](https://github.com/DataDog/synthetics-test-automation-circleci-orb)
+   * [Azure DevOps Extension](https://github.com/DataDog/datadog-ci-azure-devops)
 
 ### Pre-Release Process
 
 If you need to create a pre-release or releasing in a different channel here's how it works:
```
Comment (alai97) [inline README.md:159]:
```suggestion
To create a pre-release or releasing in a different channel:
```

File: README.md [hunk 12]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
-- create a PR for your feature branch with the channel branch as a base.
-- pick a version following this format `version-channel`, it can be `0.10.9-alpha` or `1-beta`...
-- update the `version` field in `package.json`
-- merge the Pull Request
-- create a [Github Release](https://github.com/DataDog/datadog-ci/releases/new?target=alpha&tag=0.10.9-alpha&prerelease=1&title=Alpha+prerelease):
-  - target the channel branch
-  - pick a tag based on your version `version-channel`
-  - check the `This is a pre-release` checkbox
-- publish the release and an action will publish it on npm
+1. Create a new branch for the channel you want to release to (`alpha`, `beta`, ...).
```
Comment (alai97) [inline README.md:161]:
```suggestion
1. Create a new branch for the channel you want to release to (`alpha`, `beta`, and more).
```

File: README.md [hunk 13]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+2. Create a PR for your feature branch with the channel branch as a base.
+3. Pick a version following this format: `version-channel`. It can be `0.10.9-alpha`, or `1-beta`, etc.
```
Comment (alai97) [inline README.md:163]:
```suggestion
3. Pick a version following this format: `version-channel`. For example, `0.10.9-alpha`, `1-beta`, and more.
```

File: README.md [hunk 14]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+4. Update the `version` field in `package.json`.
+5. Merge the Pull Request.
```
Comment (alai97) [inline README.md:165]:
```suggestion
5. Once you've received at least one approval, merge the Pull Request.
```

File: README.md [hunk 15]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+6. Create a [GitHub Release](https://github.com/DataDog/datadog-ci/releases/new?target=alpha&tag=0.10.9-alpha&prerelease=1&title=Alpha+prerelease):
+   1. Target the channel branch.
+   2. Pick a tag based on your version `version-channel`.
+   3. Check the `This is a pre-release` checkbox.
```
Comment (alai97) [inline README.md:169]:
```suggestion
   - Target the channel branch.
   - Pick a tag based on your version `version-channel`.
   - Check the `This is a pre-release` checkbox.
```

File: README.md [hunk 16]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+7. Publish the release and an action will publish it on npm.
```
Comment (alai97) [inline README.md:170]:
```suggestion
7. Publish the release and an action publish it on npm.
```

File: README.md [hunk 17]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
 
 <img src="./assets/pre-release.png" width="500"/>
 
+## More ways to install the CLI
+
+### Standalone binary (**beta**)
+
+If installing nodejs in the CI is an issue, standalone binaries are provided with [releases](https://github.com/DataDog/datadog-ci/releases). Only _linux-x64_, _darwin-x64_ (macOS) and _win-x64_ (Windows) are supported at the time. **These standalone binaries are in _beta_ and their stability is not guaranteed**. To install:
```
Comment (alai97) [inline README.md:178]:
```suggestion
If installing NodeJS in the CI is an issue, standalone binaries are provided with [releases](https://github.com/DataDog/datadog-ci/releases). _linux-x64_, _darwin-x64_ (macOS), and _win-x64_ (Windows) are supported. **These standalone binaries are in beta and their stability is not guaranteed**. 

To install:
```

File: README.md [hunk 18]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+#### Building your own container image
+
+You can build an image using the provided [Dockerfile](https://github.com/DataDog/datadog-ci/blob/master/container/Dockerfile):
+
+```sh
+cd container
+docker build --tag datadog-ci .
+```
+
+Optionally you can use the `VERSION` build argument to build an image for a specific version:
```
Comment (alai97) [inline README.md:229]:
```suggestion
Optionally, you can use the `VERSION` build argument to build an image for a specific version:
```

File: README.md [hunk 19]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+
+### Container image
+
+To run `datadog-ci` from a container, you can use the `datadog/ci` image available in Dockerhub as well as the public Amazon ECR and Google GC registries.
+
+```
+docker pull datadog/ci
+```
+
+Here's an example of how to run a command using the container and passing in the API and APP keys:
```
Comment (alai97) [inline README.md:212]:
```suggestion
This example demonstrates how to run a command using the container and passing in the API and APP keys:
```

File: README.md [hunk 20]
```diff
@@ -214,30 +143,95 @@ yarn prepack
...
+curl -L --fail "https://github.com/DataDog/datadog-ci/releases/latest/download/datadog-ci_darwin-x64" --output "/usr/local/bin/datadog-ci" && chmod +x /usr/local/bin/datadog-ci
+```
+
+#### Windows
+
+```sh
+Invoke-WebRequest -Uri "https://github.com/DataDog/datadog-ci/releases/latest/download/datadog-ci_win-x64.exe" -OutFile "datadog-ci.exe"
+```
+
+Then you can run `datadog-ci` commands normally:
```
Comment (alai97) [inline README.md:198]:
```suggestion
Then, you can run `datadog-ci` commands normally:
```

**Review by alai97 — APPROVED** [review]
Looks good, some copy suggestions!

**Review by m-rousse — APPROVED** [review]
:+1:

**Pull request merged**